ipc/observability/src/config.rs

```diff
@@ -12,6 +12,7 @@ use tracing_subscriber::filter::EnvFilter;
 #[serde_as]
 #[derive(Debug, Deserialize, Clone, Default, strum::EnumString, strum::Display)]
 #[strum(serialize_all = "snake_case")]
+#[serde(rename_all = "lowercase")]
```

Comment:
apparently `#[strum(serialize_all = "snake_case")]` only applies to and from string, not with serde.